README.md

```diff
@@ -124,44 +127,46 @@ The project can be built using [PlatformIO](https://platformio.org/). There's a
 Note: Some defines are board specific, this is noted below.
 
-| Feature Define | Description |
-| - | - |
-| ENABLE_WIFI            | Connect to WiFi |
-| INCOMING_WIFI_ENABLED  | Accepting incoming color data and commands |
-| ENABLE_WEBSERVER       | Turn on the internal webserver |
-| TIME_BEFORE_LOCAL      | How many seconds before the lamp times out and shows local content |
-| ENABLE_NTP             | Set the clock from the web |
-| ENABLE_OTA             | Accept over the air flash updates |
-
-| Hardware Specific | Description | Supported Boards |
-| - | - | - |
-| USE_M5DISPLAY          | Enable stats display on built in LCD | M5Stick-C and M5Stick-C Plus |
-| USE_OLED               | Enable stats display on built in OLED | Heltec Wifi Kit 32 |
-| USE_LCD                | Enable stats display on external ILI9341 LCD | Wrover32 |
-| USE_TFTSPI             | Enable stats display on external TTGO LCD | esp32dev |
-| ENABLE_AUDIO           | Listen for audio from the microphone and process it | M5Stick-C and M5Stick-C Plus |
-| ENABLE_REMOTE          | IR Remote Control | Requires IR Hardware |
-
-example in platformio.ini
+| Feature Define        | Description                                                        |
+| --------------------- | ------------------------------------------------------------------ |
+| ENABLE_WIFI           | Connect to WiFi                                                    |
+| INCOMING_WIFI_ENABLED | Accepting incoming color data and commands                         |
+| ENABLE_WEBSERVER      | Turn on the internal webserver                                     |
+| TIME_BEFORE_LOCAL     | How many seconds before the lamp times out and shows local content |
+| ENABLE_NTP            | Set the clock from the web                                         |
+| ENABLE_OTA            | Accept over the air flash updates                                  |
+
+| Hardware Specific | Description                                         | Supported Boards             |
+| ----------------- | --------------------------------------------------- | ---------------------------- |
+| USE_M5DISPLAY     | Enable stats display on built in LCD                | M5Stick-C and M5Stick-C Plus |
+| USE_OLED          | Enable stats display on built in OLED               | Heltec Wifi Kit 32           |
+| USE_LCD           | Enable stats display on external ILI9341 LCD        | Wrover32                     |
+| USE_TFTSPI        | Enable stats display on external TTGO LCD           | esp32dev                     |
+| ENABLE_AUDIO      | Listen for audio from the microphone and process it | M5Stick-C and M5Stick-C Plus |
+| ENABLE_REMOTE     | IR Remote Control                                   | Requires IR Hardware         |
+
+example in platformio.ini (prefix the flags with `D`, e.g. `ENABLE_WIFI=1` becomes `DENABLE_WIFI=1`)

```

Comment:
If you want to mention the `D`, then please also include the dash (`-`). `-D` is the build flag to set a #define from the command line.